app_builder/lib/app_builder/macos.ex

```diff
@@ -98,7 +98,8 @@ defmodule AppBuilder.MacOS do
         :info_plist,
         :url_schemes,
         :document_types,
-        :additional_paths
+        :additional_paths,
+        :uielement
```

Comment:
the docs are here: https://developer.apple.com/documentation/bundleresources/information_property_list/lsuielement

I didn't know how to name this option. FWIW wx calls this `OSXIsGUIApplication`. Any preferences?

Reply:
`:is_agent_app` perhaps, but no strong preference :)

Reply:
Sounds good, thanks!